org.eclipse.richbeans.test/src/org/eclipse/richbeans/test/shuffle/Example1Test.java

```diff
@@ -0,0 +1,69 @@
+package org.eclipse.richbeans.test.shuffle;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import org.eclipse.richbeans.examples.ExampleFactory;
+import org.eclipse.richbeans.examples.IShellCreator;
+import org.eclipse.swt.widgets.Display;
+import org.eclipse.swt.widgets.Shell;
+import org.eclipse.swtbot.swt.finder.widgets.SWTBotLabel;
+import org.eclipse.swtbot.swt.finder.widgets.SWTBotStyledText;
+import org.junit.Test;
+
+/**
+ * 
+ * Basic test running all the examples.
+ * 
+ * @author Matthew Gerring
+ *
+ */
+public class Example1Test extends ShellTest {

```

Comment:
If you wanted to do that you could use junit "rules". Because there are before and after in the super class, you could probably use an ExternalResource rule. Not sure how it works with before and after class.